terraform-aws-github-runner/modules/runners/lambdas/runners/src/scale-runners/scale-down.ts

```diff
@@ -387,6 +387,26 @@ export async function isEphemeralRunner(ec2runner: RunnerInfo, metrics: ScaleDow
   return runnerTypes.get(ec2runner.runnerType)?.is_ephemeral ?? false;
 }
 
+export async function minRunners(ec2runner: RunnerInfo, metrics: ScaleDownMetrics): Promise<number> {
+  if (ec2runner.runnerType === undefined) {
+    return Config.Instance.minAvailableRunners;
+  }
+
+  const repo: Repo = (() => {
+    if (Config.Instance.enableOrganizationRunners) {
+      return {
+        owner: ec2runner.org !== undefined ? (ec2runner.org as string) : getRepo(ec2runner.repo as string).owner,
+        repo: Config.Instance.scaleConfigRepo,
+      };
+    }
+    return getRepo(ec2runner.repo as string);
+  })();
```

Comment:
Why does this have to be put inside a temporary lambda that's immediately invoked?

Reply:
I'm a Typescript newbie so I'm not really sure myself, I used another function as a template that I copied from. I'll work on removing the temporary lambda function.

Reply:
So the isEphemeralRunner() function above this uses the same code which I copied from. I'm inclined to keep it the same since it works. Maybe we can update both functions as an improvement in a separate pull request if we want to unravel the lambda.